### ISSUE TITLE
train my own datasets,but the loss: cls always be zero

### ISSUE THREAD

**Issue by daixiaoxi** [marvis/pytorch-yolo2#100]
I tried to train my own dataset, concluding only one class. the cls always be zero ,anyone knows the reason?

Thanks a lot 

64: nGT 661, recall 32, proposals 34, loss: x 135.139847, y 92.881287, w 1644.332642, h 1461.494873, conf 6513.747070, cls 0.000000, total 9847.595703
128: nGT 696, recall 34, proposals 246, loss: x 131.766190, y 91.255302, w 1275.351196, h 1089.764282, conf 5960.305176, cls 0.000000, total 8548.442383
178: nGT 526, recall 39, proposals 93, loss: x 94.376816, y 70.759315, w 935.032471, h 897.536560, conf 3918.665771, cls 0.000000, total 5916.371094
